### PR TITLE
BI-658 Remodel index/front page

### PR DESCRIPTION
**Pull request by eawoods**
The front page needed some work, both in design and in usefulness of content -- especially for the case of the public sandbox.

I made a few changes to the design that had been posted on #ui-design on Nov 5th, most notably to move the images from a vertical bar to a horizontal one. They worked better with narrower screens and are easier to control as the content changes shape responsively.

**There is currently only one sidebar, containing the guest account information for the public sandbox.** If we want to add different content for the coordinator sandbox or dev purposes, that's easy enough to do if needed.

## Technical Changes

Layout template structure now has the nav element outside of `BaseSideBarLayout`. This is to more accurately describe the function of the sidebar for accessibility purposes, since the index page is using the sidebar for content other than navigation.

## Further Enhancements

Chris and I talked about the idea of restructuring the templates somewhat to be more flexible, and to make the login button and related modals a component. This is deemed out of scope for the current task.

## Screenshots

Desktop version
Screen Shot 2020-11-19 at 10 25 06 AM

Mobile version
Screen Shot 2020-11-19 at 10 24 12 AM

### CODE REVIEW

**Review by nickpalladino — COMMENTED** [review]
Looks good. Maybe at some point we can make this tied to only the public sandbox mode and have something else for other modes. Had a couple other comments. Also can't remember if the page filled the whole view before but my footer is a bit above the bottom of the page.

File: src/views/Index.vue [hunk 1]
```diff
@@ -16,222 +16,110 @@
...
+            Please note:
+          </p>
+          <ul>
+            <li>All submitted data are public; do not submit private data.</li>
+            <li>The database is routinely reset and submitted data may be removed without notice.</li>
+          </ul>
+          <p>We encourage you to share your experiences using the sandbox with us!</p>
+          <p class="has-text-centered">
+            <a class="button is-primary">
+              Send feedback to our team
```
[the remark this answers is not in the document]
Comment (eawoods) [reply src/views/Index.vue:51]: oops, yes I did mean to put that in. Will fix!

File: src/components/layouts/InfoSideBarLayout.vue [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        <div class="pl-3">
+          <h1
+            class="title has-text-warning is-5"
+          >
+            Guest Accounts
+          </h1>
+          <p class="has-text-light is-paddingless">
+            Use these credentials to explore sample data and currently available features.
+          </p>
+          <div class="is-hidden-touch">
```
[the remark this answers is not in the document]
Comment (eawoods) [reply src/components/layouts/InfoSideBarLayout.vue:41]: Yeah, it seemed redundant to have the login button so close to the login button that's in the header - on a mobile device it just looked weird to have both buttons.

**Review by timparsons — APPROVED** [review]
Looks great!!

File: src/components/layouts/InfoSideBarLayout.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (nickpalladino) [reply src/components/layouts/InfoSideBarLayout.vue:41]: gotcha, ya probably under the header.

File: src/views/Index.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (eawoods) [reply src/views/Index.vue:51]: Changed and pushed

**Comment by eawoods** [comment]
I went ahead and added some perfunctory text for coordinator and default modes in the sidebar, to prevent confusion.
@nickpalladino let's debug that footer, I am seeing it stuck to the bottom but if you are not seeing the same let's fix it.

**Review by nickpalladino — APPROVED** [review]
One minor thing I noticed with the footer is that the contents aren't vertically centered due to a mb-4 which I'm guessing is from the version stuff Tim was working on.

**Comment by eawoods** [comment]
Footer not vertically centered: yeah probably. At some point maybe a "UI/CSS cleanup" card needs to happen...sounds like a just-before-a-holiday kind of task to me. :)

**Pull request merged**